src/base.jl

```diff
@@ -974,11 +926,19 @@ Return a TimeSeries.TimeArray for the requested time series parameters.
 function get_time_series_array(
     ::Type{T},
     component::Component,
-    initial_time::Dates.DateTime,
-    label::AbstractString,
-    horizon::Union{Nothing, Int} = nothing,
+    name::AbstractString;
+    start_time::Union{Nothing, Dates.DateTime} = nothing,
+    len::Union{Nothing, Int} = nothing,
+    count::Int = 1,
 ) where {T <: TimeSeriesData}
-    return IS.get_time_series_array(T, component, initial_time, label, horizon)
+    return IS.get_time_series_array(
```

Comment:
Do we need to implement this for Deterministic?

Reply:
I think this supports Deterministic as it is.  Why do you disagree?

Reply:
I am just inquiring to see if we need to test it.

Reply:
We should.  Can you add some test data to PowerSystems or PowerSystemsTestData?

Reply:
Ok, I will make sure to make/add data for this test. I am almost sure that the 5-bus case already has data on this effect.